src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -109,6 +114,11 @@ object SparkCassandraStress {
           ("spark.cassandra.input.split.size" -> arg.toString))
       } text {"Read input size"}
 
+      opt[Int]('u',"throughput") optional() action { (arg,config) =>
+        config.copy(sparkOps = config.sparkOps +
+          ("spark.cassandra.output.throughput_mb_per_sec" -> arg.toString))
```

Comment:
I think all the ones you specified can have defaults and be part of this code, I just meant any parameters that adjust the spark connector through the SparkConf should be removed since there is already a way to set these without having to memorize all the letters :)